packages/dependency-extraction-webpack-plugin/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "@wordpress/dependency-extraction-webpack-plugin",
+	"version": "1.0.0",
+	"description": "Extract WordPress script dependencies from webpack bundles.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"webpack",
+		"dependency"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/dependency-extraction-webpack-plugin/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/dependency-extraction-webpack-plugin"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"files": [
+		"index.js"
+	],
+	"main": "index.js",
+	"dependencies": {
+		"@wordpress/scripts": "file:../scripts",
+		"webpack": "4.8.3",
+		"webpack-sources": "1.3.0"
```

Comment:
It's fine to use ranges for `webpack-sources`. I'm not quite sure why we didn't use ranges for `webpack` as well. We should fix it when we migrate to v5.0.

Reply:
I've added `^` to the versions, is that sufficient or did you have something else in mind?